lib/msal-browser/package.json

```diff
@@ -31,10 +32,7 @@
     "test": "test"
   },
   "files": [
-    "dist",
-    "lib/msal-browser.js",
-    "lib/msal-browser.js.map",
-    "lib/msal-browser.min.js"
```

Comment:
I added the cdn builds to npm so that if folks want the cdn builds, they can get them from npm. It shouldn't impact folks consuming non-cdn builds.

Reply:
Gotcha, I'll put this back then.